### PR TITLE
chore: add deploy config for l2 protocol governance

### PR DESCRIPTION
**Pull request by kangsorang**
# Description

Deploy config added - l2GovernorVotingPeriodBlocks.

Seperate transfetOwnership action to hardhat task.

Rollback the change in https://github.com/kroma-network/kroma/pull/207, which is related to mainnet config timeLockMinDelaySeconds. This value will be managed separately.

### CODE REVIEW

**Review by 0xHansLee — APPROVED** [review]
Lgtm. Please check sara's comment.

File: packages/contracts/tasks/transfer-ownership.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    scTokenProxyAddress
+  )
+  const currentScTokenOwner = await scToken.owner()
+  if (
+    hre.ethers.utils.getAddress(currentScTokenOwner) ===
+    hre.ethers.utils.getAddress(newOwner)
+  ) {
+    console.log(
+      'skip the SecurityCouncilToken owner transfer process because the owner has already been transferred.'
+    )
```
Comment (seolaoh) [inline packages/contracts/tasks/transfer-ownership.ts:70]: As I mentioned in previous comment, the owner has been changed to same address. We need to return this function when `newOwner` address is same as `currentScTokenOwner`.
```suggestion
    )
    return
```

File: packages/contracts/src/deploy-config.ts [hunk 2]
```diff
@@ -200,10 +200,16 @@ interface RequiredDeployConfig {
...
+  /**
+   * L2 : Governor voting period in block.
+   */
+  l2GovernorVotingPeriodBlocks: number
+
   /**
    * Initial minimum delay for operations.
    */
   timeLockMinDelaySeconds: number
+
```
Comment (seolaoh) [inline packages/contracts/src/deploy-config.ts:212]:
```suggestion
```

**Review by seolaoh — APPROVED** [review]
LGTM

**Comment by seolaoh** [comment]
I force pushed another commit in mistake. Just reverted it, so please ignore it.

**Review by Pangssu — APPROVED** [review]

**Pull request merged**